### PR TITLE
Päivän käyttöastegraafin suunnitelma

### PR DESCRIPTION
**Pull request by Joosakur**
Visualisoidaan olemassa olevan raportin data

Screenshot 2024-08-26 at 16 22 55

### CODE REVIEW

File: frontend/src/employee-frontend/components/unit/tab-unit-information/occupancy/OccupancyDayGraphPlanned.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+      childrenPresent: number
+      staffRequired: number
+    }
+  }>({})
+  const [tooltipVisible, setTooltipVisible] = useState<boolean>(false)
+
+  const showTooltip = useCallback(() => {
+    if (!tooltipVisible) {
+      setTooltipVisible(true)
+    }
```
Comment (akheron) [inline frontend/src/employee-frontend/components/unit/tab-unit-information/occupancy/OccupancyDayGraphPlanned.tsx:82]: The `if` is unnecessary. `setTooltipVisible(true)` does nothing if it's already true.

Comment (akheron) [reply frontend/src/employee-frontend/components/unit/tab-unit-information/occupancy/OccupancyDayGraphPlanned.tsx:82]: We also already have a `useBoolean` helper that exposes callbacks like `toggle`, `on` and `off`.

Comment (Joosakur) [reply frontend/src/employee-frontend/components/unit/tab-unit-information/occupancy/OccupancyDayGraphPlanned.tsx:82]: Copy-pasted without reading :D I'll fix both

Comment (Joosakur) [reply frontend/src/employee-frontend/components/unit/tab-unit-information/occupancy/OccupancyDayGraphPlanned.tsx]: https://github.com/espoon-voltti/evaka/pull/5565/commits/cf07f560911655535a236f563179955cb38bce3f

**Review by akheron — APPROVED** [review]

**Pull request merged**